### PR TITLE
⌚ If prices.tf goes down, sync prices.

### PR DESCRIPTION
**Pull request by brysondev**
I hope this works a bit better than before.

Basically, if prices.tf goes down at any point, we redownload the pricelist.

This prevents any syncing issues down the line, I hope.

### CODE REVIEW

File: app/lib/pricer/pricestf/prices-tf-api.js [hunk 1]
```diff
@@ -50,21 +50,12 @@ class PricesTfApi {
                 ...headers,
             });
         } catch (e) {
-            if (e && 401 === e['statusCode']) {
```
Comment (arik123) [inline app/lib/pricer/pricestf/prices-tf-api.js:53]: you should have kept this in, we need to reauthenticate

Comment (brysondev) [reply app/lib/pricer/pricestf/prices-tf-api.js]: Ok added it back!

**Comment by arik123** [comment]
What if this happens during getting pricelist ? It seems to me that it would just try to get it indefinitelly

**Comment by brysondev** [comment]
> What if this happens during getting pricelist ? It seems to me that it would just try to get it indefinitelly

Yes, it will keep trying until i can successfully refresh the pricelist in its entirety. Should I have it like I did previously for now?

**Review by arik123 — APPROVED** [review]
I think I like it now, will merge it later

**Review by idinium96 — APPROVED** [review]
Thanks for the PR, looks good and I am merging now.

**Pull request merged**